core/docs/spark-qualification-tool.md

```diff
@@ -32,50 +20,70 @@ applicable environments.  Here are the cluster information for the ETL benchmark
 | Databricks AWS   | 8x m6gd.8xlage    | 8x g5.8xlarge                  |
 | Databricks Azure | 8x E8ds_v4        | 8x NC8as_T4_v3                 |
 
-Note that all benchmarks were run using the [NDS benchmark](https://github.com/NVIDIA/spark-rapids-benchmarks/tree/dev/nds) at SF3K (3 TB).
-
 > **Disclaimer!**  
-> Estimates provided by the Qualification tool are based on the currently supported "_SparkPlan_" or "_Executor Nodes_"
-> used in the application. It currently does not handle all the expressions or datatypes used.  
-> Please refer to "[Understanding Execs report](#execs-report)" section and the
-> "[Supported Operators](https://github.com/NVIDIA/spark-rapids/blob/main/docs/supported_ops.md)" guide to check the types and expressions you are using are supported.
+> Estimates provided by the Qualification tool assume the application is run on a dedicated cluster where it can use all available Spark resources. The estimates are based on the "_SparkPlan_" or "_Executor Nodes_"
+> used in the Spark application and currently supported by the RAPIDS Accelerator for Apache Spark. Not all expressions and datatypes are currently supported,
+> Please refer to the "[Understanding Execs report](#execs-report)" section and the
+> "[Supported Operators](https://github.com/NVIDIA/spark-rapids/blob/main/docs/supported_ops.md)" guide to verify the types and expressions used are supported.
+
+## Table of Contents
+
+- [Running the Qualification Tool](#Running-the-Qualification-Tool)
+   - [Standalone on Spark event logs for CSP environments](#running-the-standalone-qualification-tool-for-csp-environments-on-spark-event-logs)
+   - [Standalone on Spark event logs for other environments](#running-the-qualification-tool-standalone-on-spark-event-logs)
+   - [Inside a running Spark application using the API](#running-the-qualification-tool-inside-a-running-spark-application-using-the-api)
+   - [On active jobs using a Spark Listener](#running-the-qualification-tool-using-a-spark-listener)
+- [Understanding the Qualification Tool Output](#understanding-the-qualification-tool-output)
+   - [Application Report](#application-report)
+   - [Stages Report](#stages-report)
+   - [Execs Report](#execs-report)
+   - [MLFunctions Report](#mlfunctions-report)
+   - [MLFunctions Total Duration Report](#mlfunctions-total-duration-report)
+   - [HTML Report](#html-report)
+- [Example Reports and File Output](#example-reports-and-file-output)
+- [Compiling the Tools Jar from Source](#compiling-the-tools-jar-from-source)
+- [S3 File Path Support and Configuration](#s3-file-path-support-and-configuration)
+- [Tools Jar Maven Dependency](#qualification-tool-maven-dependency-snippet)
```

Comment:
Why are we replacing the auto-generated TOC `{:toc}` with a manual one?
A manual TOC is hard to maintain because it will go out of sync as changes keep coming.